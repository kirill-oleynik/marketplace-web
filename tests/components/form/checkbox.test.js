const React = require('react');
const renderer = require('react-test-renderer');
const Checkbox = require('../../../app/components/form/checkbox').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const tree = renderer.create(
      <Checkbox name="test" text="Test" checked={false} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
