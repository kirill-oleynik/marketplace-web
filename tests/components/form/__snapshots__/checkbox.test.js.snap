// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<div>
  <label
    className="checkbox"
    htmlFor="checkbox-test"
  >
    <input
      checked={false}
      className="checkbox__input"
      id="checkbox-test"
      name="test"
      type="checkbox"
    />
    <span
      className="checkbox__icon icon"
    />
    <span
      className="checkbox__text"
    >
      Test
    </span>
  </label>
</div>
`;
